airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/spec.yaml

```diff
@@ -0,0 +1,15 @@
+documentationUrl: https://developer.zendesk.com/api-reference/sales-crm/introduction/#get-up-to-speed-with-the-sell-api
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Source Zendesk Sell Spec
+  type: object
+  required:
+    - api_token
+  properties:
+    api_token:
+      title: API token
+      type: string
+      description: "The API token for authenticating to Zendesk Sell"
+      examples: 
+        - "f23yhd630otl94y85a8bf384958473pto95847fd006da49382716or937ruw059"
+      airbyte_secret: true
```

Comment:
I think would be great to add the `start_date` parameter so users can choose since when they want to sync data.

Reply:
For incremental syncs only, right?